cve_bin_tool/checkers/sqlite.py

```diff
@@ -166,8 +175,8 @@ def get_version_map():
             version_map.append([last_version, id_match.group(1)])
             print('        [\"' + last_version + '", "' + id_match.group(1) + '"],')
 
-
     return version_map
 
+
```

Comment:
nit: Please don't add newlines, we should stay at a single blank line between blocks throughout.